src/xcode/ENA/ENA/Source/Services/HealthCertificate/Models/TestCertificateRequest.swift

```diff
@@ -55,6 +58,7 @@ class TestCertificateRequest: Codable, Equatable {
 		encryptedDEK = try container.decodeIfPresent(String.self, forKey: .encryptedDEK)
 		encryptedCOSE = try container.decodeIfPresent(String.self, forKey: .encryptedCOSE)
 		requestExecutionFailed = try container.decode(Bool.self, forKey: .requestExecutionFailed)
+		labId = try container.decode(String.self, forKey: .labId)
```

Comment:
```suggestion
		labId = try container.decodeIfPresent(String.self, forKey: .labId)
```

Just for downward compatibility, not super important as this class was not released yet, but the developers and testers might appreciate it.